reportportal/v5/README.md

```diff
@@ -406,10 +406,15 @@ minio:
   installdep:
     enable: false
   endpoint: http://<minio-release-name>.default.svc.cluster.local:9000
+  region: null
```

Comment:
do not pass "null" as string, use empty string or remove the variable at all

Reply:
set as empty